mappings/net/minecraft/network/message/MessageChain.mapping

```diff
@@ -0,0 +1,66 @@
+CLASS net/minecraft/class_7610 net/minecraft/network/message/MessageChain
+	COMMENT A class for handling the "message chain".
+	COMMENT
+	COMMENT <p>{@link MessageHeader} includes the signature of the last message the client has seen.
+	COMMENT This can be used to verify the legitimacy of a chain of messages, since if the chain
+	COMMENT is valid, the last message's  "previous signature" should be able to verify the preceding
+	COMMENT message.
+	COMMENT
+	COMMENT <p>Clients signing a message with its preceding message's signature is called
+	COMMENT "packing", and the server creating a signed message with its preceding message's
+	COMMENT signature is called "unpacking". Unpacked messages can then be verified to check the
+	COMMENT chain's legitimacy.
+	FIELD field_39818 precedingSignature Lnet/minecraft/class_7469;
+	METHOD method_44874 getUnpacker ()Lnet/minecraft/class_7610$class_7611;
+	METHOD method_44875 pack (Lnet/minecraft/class_7501;Lnet/minecraft/class_7470;Lnet/minecraft/class_2561;)Lnet/minecraft/class_7610$class_7613;
+		ARG 1 signer
+		ARG 2 newMetadata
+		ARG 3 newContent
```

Comment:
Why do these have a "new" prefix? On the other methods too

Reply:
Maybe we can drop it.